packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseDashboardFooter.test.tsx

```diff
@@ -57,13 +57,15 @@ describe('ReleaseDashboardFooter', () => {
   })
 
   describe('for a published release', () => {
-    test('shows revert button for asap release', async () => {
+    // Revert button is now hidden in the UI until feature flag "releases plus" is added - see ReleasesRevert component
```

Comment:
On the story where we rework this / add to this we should make sure to mention we have these tests being skipped